File: app/src/main/java/me/rosuh/android/jianews/storage/IDataModel.java
package me.rosuh.android.jianews.storage;

import java.util.List;

import me.rosuh.android.jianews.bean.ArticleBean;

/**
 * @author rosu
 * @date 2018/9/30
 *
 * Presenter 需要实现的数据接口
 */
public interface IDataModel {
    void onInfo(int info);

    void onDataResponse(List<ArticleBean> articleBeanList);
}

File: app/src/main/java/me/rosuh/jianews/storage/IDataModel.kt
package me.rosuh.jianews.storage

import me.rosuh.jianews.bean.ArticleBean

/**
 * @author rosu
 * @date 2018/9/30
 *
 * Presenter 需要实现的数据接口
 */
interface IDataModel {

    fun onInfo(info: Int)

    fun onDataResponse(articleBeanList: List<ArticleBean>)
}
